src/test/java/productHistoricalTest/LoginSignupCompanyTest.java

```diff
@@ -1,12 +1,7 @@
 package productHistoricalTest;
 
 import baseClassPackage.BaseClasses;
-import productHistorical.AddProduct1Page;
-import productHistorical.AddProduct4Page;
 import productHistorical.LoginSignupCompanyPage;
-
-import java.io.IOException;
-
 import org.openqa.selenium.WebDriver;
 import org.testng.annotations.BeforeClass;
 import org.testng.annotations.Test;
```

Comment:
@mayankchhajer Please do not pass any argument in the verifyConfirmationMsg()  message, Please revert your changes .